src/errors.rs

```diff
@@ -72,6 +72,7 @@ impl error::Error for ArrayError {
 pub enum ReceiveError {
     /// A plugin submitted a field that contained invalid UTF-8 characters
     Utf8(String, &'static str, Utf8Error),
+    Metadata(String, String, &'static str),
```

Comment:
Oof, thanks for following the convention, I'll need to apply some best practices in the future and name these fields 😄